cylc/flow/run_modes/__init__.py

```diff
@@ -40,7 +40,7 @@
     ]
 
 
-class RunMode(Enum):
+class RunMode(str, Enum):
```

Comment:
I'm not sure I understand subclassing `str`

Reply:
```python
RunMode('Live') == 'Live'
True
```

Reply:
I'm a bit torn on this. I see its convenience, but also I think leaning too heavily on the convenience could result in problems. For example, your example doesn't work!
```
In [1]: from cylc.flow.run_modes import RunMode

In [2]: RunMode('Live') == 'Live'
Out[2]: False
```
Because the case doesn't match the enum's value.

Without subclassing str, normally Mypy will flag this is a non-overlapping equality check.

Reply:
Don't worry is't not a big deal (i.e. I'm happy to get rid.)